vignettes/ees-api-workflow.Rmd

```diff
@@ -144,12 +143,20 @@ eesyapi::get_meta(dataset_id = "7c0e9201-c7c0-ff73-bee4-304e731ec0e6") |>
 ```
 
 ### Geographies
+
 ``` {r}
 eesyapi::get_meta(dataset_id = "7c0e9201-c7c0-ff73-bee4-304e731ec0e6") |>
   magrittr::extract2("locations") |>
   dplyr::filter(label %in% c("York", "England"))
 ```
 
+### Filter columns
+
+``` {r}
+eesyapi::get_meta(dataset_id = "7c0e9201-c7c0-ff73-bee4-304e731ec0e6") |>
+  magrittr::extract2("filter_columns")
+```
+
 ### Filter items
 
```

Comment:
It's kind of obvious given the two code outputs, but might still be worth adding a line here saying that the col_ids in the filter items query correspond to the col_ids in the filter columns query and that it's the item_ids that they need for creating data queries. I know you've got a paragraph on it lower down, but think a short line on it here next to the output would be useful as well.

Reply:
Yeah good idea - have added